chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -315,6 +315,12 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
 
         // Start chat from SDK Event
         BroadcastService.getMessageByEventName(BroadcastEvent.StartChat).subscribe((msg: ICustomEvent) => {
+            // If chat is out of operating hours chat widget sets the conversation state to OutOfOffice.
+            if (props?.chatConfig?.LiveWSAndLiveChatEngJoin?.OutOfOperatingHours.toLowerCase() === "true") {
+                state?.appStates.isMinimized && dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: false });
```

Comment:
it needs to load the Oout of office hours pane

Reply:
Sure.
**it needs to load the Out of office hours pane** - I will check with you for OOOH pane. I thought OOOH message that gets displayed is pane.
**test in popout / normal** - tested / passed
**test refreshing** - tested / passed
**test in multitab** - Scenario is not clear to me